cmd/neofs-adm/internal/modules/morph/dump_hashes.go

```diff
@@ -203,8 +208,9 @@ func printContractInfo(cmd *cobra.Command, infos []contractDumpInfo) {
 		if info.version == "" {
 			info.version = "unknown"
 		}
-		_, _ = tw.Write([]byte(fmt.Sprintf("%s\t(%s):\t%s\n",
-			info.name, info.version, info.hash.StringLE())))
+		_, _ = tw.Write([]byte(fmt.Sprintf("%s\t(%s):\t%s\t%s\n",
+			info.name, info.version, info.hash.StringLE(),
+			time.UnixMilli(info.expiration).String())))
```

Comment:
`fillContractExpiration` ignores errors so here we can see zero. Will the user understand this output?

Reply:
Do you propose removing it completely in this case? Now it's like `1970-01-01 03:00:00 +0300 MSK`

Reply:
Yeah, strictly speaking I think `1970-01-01 03:00:00 +0300 MSK` can be misinterpreted as "expired 50 years ago". Do not show value at all or print `unlimited` would be clearer for user IMO

Reply:
Fixed.